PaddleCV/object_detection/ppdet/data/data_feed.py

```diff
@@ -247,9 +262,13 @@ def __init__(self,
                  dataset_dir=VOC_DATASET_DIR,
                  annotation=VOC_TEST_ANNOTATION,
                  image_dir=VOC_IMAGE_DIR,
+                 image_or_dir=VOC_IMAGE_DIR,
+                 image_ext=None,
                  use_default_label=VOC_USE_DEFAULT_LABEL):
```

Comment:
Change all VOC_xx in SimpleDataSet to None?

Reply:
Done